BPL-CBH LCCN

```diff
@@ -0,0 +1,89 @@
+'MacroName:BPLLCCN
```

Comment:
How about simply LCCN as a name?
Connexion will display it as Brooklyn!LCCN when user mouse over the shortcut.